components/post_view/post/post.tsx

```diff
@@ -168,20 +179,20 @@ class Post extends React.PureComponent {
         this.props.actions.selectPost(post);
     }
 
-    handleCardClick = (post) => {
+    handleCardClick = (post: IPost) => {
         if (!post) {
             return;
         }
         this.props.actions.selectPostCard(post);
     }
 
-    handlePostClick = (e) => {
+    handlePostClick = (e: React.MouseEvent<HTMLDivElement, MouseEvent>) => {
         const post = this.props.post;
         if (!post) {
             return;
         }
 
-        if (this.props.channelIsArchived || post.system_post_ids) {
+        if (this.props.channelIsArchived) {
```

Comment:
Why did the conditional logic here change to not include `post.system_post_ids`?